### ISSUE TITLE
Expression extractor does not find templatesUrls in 'web' directory

### ISSUE THREAD

**Issue by rkirov** [angular/angular.dart#1623]
Sample app:
web/main.dart
```
@Component(
  templateUrl: 'myTemplate.html'
```
myTemplate.html is in `web/` but the transformer looks for it in package root.

```
pub build
> line 11, column 1 of web/main.dart: Unable to find mytmp.html at shadow_demo|mytmp.html
```

This manifests rather nastily during runtime as a `Missing getter: (o) => o.foo` error.

**Comment by naomiblack** [comment]
@rkirov can we move this over to the wiki instead of an issue? :)

**Comment by rkirov** [comment]
Will be closed by #1652.
